.github/workflows/build-macos-x86_64.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     name: build ${{ inputs.version }} on macOS x86_64
     runs-on: macos-latest
```

Comment:
macos-13 (macos-latest is arm64 now)

Reply:
This was already fixed in #427, which has been merged, so this should be ok.